pynopegl-utils/pynopegl_utils/qml/ngl_widget.py

```diff
@@ -52,7 +52,11 @@ def createRenderer(self):
         return self._renderer
 
     def set_scene(self, scene):
-        model_data = livectls.get_model_data(scene)
+        if scene:
```

Comment:
Why is this useful while we have the stop method ?

Reply:
Mmh yeah I thought I still had something like that but that commit is not needed anymore so dropped